velox/substrait/tests/VeloxSubstraitRoundTripPlanConverterTest.cpp

```diff
@@ -112,3 +112,41 @@ TEST_F(VeloxSubstraitRoundTripPlanConverterTest, values) {
 
   assertPlanConversion(plan, "SELECT * FROM tmp");
 }
+
+TEST_F(VeloxSubstraitRoundTripPlanConverterTest, aggragateCount) {
+  auto vectors = makeVectors(2, 7, 3);
+  createDuckDbTable(vectors);
+
+  auto plan = PlanBuilder()
+                  .values(vectors)
+                  .filter("c6 < 24")
+                  .aggregation(
```

Comment:
use singleAggregation for readability

Let's also add test for final and intermediate aggregations as well as global aggregations and aggregations with masks.

Reply:
done.